src/main/java/seedu/duke/account/Account.java

```diff
@@ -0,0 +1,297 @@
+package seedu.duke.account;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+import seedu.duke.*;
+import seedu.duke.exception.FinanceException;
+import seedu.duke.exception.FinanceException.ExceptionCollection;
+
+public class Account {
+    protected Wallet wallet;
+    protected String loginTime;
+
+    public Account(Wallet wallet) {
+        this.wallet = wallet;
+        DateTimeFormatter dtf = DateTimeFormatter.ofPattern("uuuu-MM-dd>HH:mm:ss");
+        loginTime = dtf.format(LocalDateTime.now());
+    }
+
+    public boolean handleAccountRequest() throws FinanceException {
```

Comment:
This method is too long. It's difficult to read and reason about the control flow of the program.